src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/sizes/parameternumber/Example3.java

```diff
@@ -0,0 +1,41 @@
+/*xml
+<module name="Checker">
+  <module name="TreeWalker">
+    <module name="ParameterNumber">
+      <property name="ignoreOverriddenMethods" value="true"/>
+    </module>
+  </module>
+</module>
+*/
+
+package com.puppycrawl.tools.checkstyle.checks.sizes.parameternumber;
+
+// xdoc section -- start
+class Example3 extends ExternalService3 {
+
+  @JsonCreator
+  // violation below, 'More than 7 parameters (found 8)'
+  Example3(int a, int b, int c, int d,
+           int e, int f, int g, int h) {}
+
+  // violation below, 'More than 7 parameters (found 8)'
+  Example3(String a, String b, String c, String d,
+           String e, String f, String g, String h) {}
+
+  @interface JsonCreator {}
```

Comment:
Please move it below `// xdoc section -- end`, user should not pay attention to this declaration.

Reply:
done. any other changes needed ?
I'm not sure why the links are not working either. I had my friend try it and she were able to open them.